lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerLaunchTests.cs

```diff
@@ -77,22 +77,20 @@ public class PuppeteerLaunchTests : PuppeteerBaseTest
             }
         }
 
-        [Fact(Skip = "https://github.com/kblok/puppeteer-sharp/issues/76")]
+        [Fact]
         public async Task ShouldRejectAllPromisesWhenBrowserIsClosed()
         {
             using (var browser = await Puppeteer.LaunchAsync(
                 TestConstants.DefaultBrowserOptions(),
                 TestConstants.ChromiumRevision))
             using (var page = await browser.NewPageAsync())
             {
-                var neverResolves = page.EvaluateFunctionHandleAsync("() => new Promise(r => {})");
-                await browser.CloseAsync();
-
-                await neverResolves;
-                var exception = await Assert.ThrowsAsync<Exception>(() => neverResolves);
+				var neverResolves = page.EvaluateFunctionHandleAsync("() => new Promise(r => {})");
```

Comment:
@Meir017 just if you want you to jump in and give me a hand in here.
`neverResolves` is not being canceled when we run all the tests altogether. But it is being resolved when I debug it. That task gets stuck in `WaitingForActivation`. It's diving me crazy.

Reply:
The `callback.TaskWrapper.SetException` on Session.OnClosed is being called but that exception never gets to the test. It's like someone is cutting the wire in the middle.